### DISCUSSION TITLE
alternative way to have notes disappear in editor timeline

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#18407] [Ideas] [answered]
https://user-images.githubusercontent.com/78441708/170164533-f598c220-8c21-4ac6-8110-575e24bccb0b.mp4

https://user-images.githubusercontent.com/78441708/170164539-41d9b459-5d9e-44ce-8275-cf0d8c0819d0.mp4

right now, the way notes disappear is awkward and borderline buggy. i propose you make sure items completely clear the screen before removing them, instead of removing them when they get close to the edge.

one of the recordings the resolution is smaller, to show that they disappear closer to the edge on smaller resolutions, which further makes this weird. 

if we could also bundle in customizable hit animations in the editor (change type [2022.509 & previous vs 2022.514 and onwards] and optimally opacity changes and size or length of display) that would be great.

**Comment by peppy** [comment]
99% sure a tracking issue exists for this. It's a known bug anyway. Zoom in to resolve for now.

**Comment by frenzibyte** [comment, marked as answer]
Should be resolved in https://github.com/ppy/osu/pull/18462
